servers/src/main/java/tachyon/worker/block/BlockDataManager.java

```diff
@@ -79,15 +80,14 @@
   /**
    * Creates a BlockDataManager based on the configuration values.
    *
-   * @param tachyonConf the configuration values to use
    * @param workerSource object for collecting the worker metrics
    * @throws IOException if fail to connect to under filesystem
    */
-  public BlockDataManager(TachyonConf tachyonConf, WorkerSource workerSource)
+  public BlockDataManager(WorkerSource workerSource)
       throws IOException {
+    mTachyonConf = WorkerContext.getConf();
```

Comment:
Not something this PR needs to address, but does it still make sense to assign `WorkerContext.getConf()` to a variable?